src/Microsoft.OData.Core/ODataError.cs

```diff
@@ -4,16 +4,23 @@
 // </copyright>
 //---------------------------------------------------------------------
 
+
+
 namespace Microsoft.OData.Core
 {
     #region Namespaces
     using System;
     using System.Collections.Generic;
     using System.Diagnostics;
     using System.Diagnostics.CodeAnalysis;
+    using System.Globalization;
     using System.Linq;
+    using System.Text;
+
+    using Microsoft.OData.Core.Json;

```

Comment:
ODataError is a high-level class that should not have a dependency on a particular format.